modules/mco-update-boot-images.adoc

```diff
@@ -1,156 +0,0 @@
-// Module included in the following assemblies:
```

Comment:
Module was used by Nodes only. No longer needed with the promotion of the topic to its own assembly.